modules/exploits/linux/http/panos_management_unauth_rce.rb

```diff
@@ -0,0 +1,197 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
```

Comment:
Can we register a Writable Dir datastore option for this module?

Reply:
addressed via commit [d2f6e0e](https://github.com/rapid7/metasploit-framework/pull/19663/commits/d2f6e0e10fbad82ddd5e12b908299d2caa008d54)